keps/sig-auth/3257-cluster-trust-bundles/README.md

```diff
@@ -563,6 +579,64 @@ For example, if I maintain `example.com/my-signer`, I can use the following stra
   canary object first, and assess the health of the canary workloads.
 * Once I am satisfied that the change is safe, I edit the live object.
 
+### Publishing the kube-apiserver-serving Trust Bundle
+
+Today, the trust bundle that allows verifying kube-apiserver serving certificate(s)
+at its internal endpoints is distributed into every namespace using a configMap.
+This is so that it can be mounted along with the ServiceAccount token in order
+for the workloads to be able to communicate with the kube-apiserver.
+
+In the future, we should be able to replace mounting these configMaps in pods for
+for kube-apiserver trust with the projected volume from this feature, and so a
+ClusterTrustBundle API object will be created for all clusters:
+
+```yaml
+apiVersion: v1beta1
+kind: ClusterTrustBundle
+metadata:
+  generateName: kubernetes.io:kube-apiserver-serving:
```

Comment:
Why not just `name`?

Reply:
@liggitt proposed not to commit to a name, but rather a signer name, which I thought made sense.